api/src/main/java/jakarta/resource/spi/security/GenericCredential.java

```diff
@@ -53,7 +53,7 @@
  *  is via the <code>org.ietf.jgss.GSSCredential</code> interface in 
  *  J2SE Version 1.4, which provides similar functionality.
  */
-
+@Deprecated
```

Comment:
When deprecating, you should tell the user what you expect them to use instead.

Reply:
I do not deprecate anything, see `@javadoc` in javadoc. What do you want the person who just follows compiler advice to add there?